### ISSUE TITLE
Training with Pretrained Model, keep and increase accuracy old class and add new class but asking a more efficient way.

### ISSUE THREAD

**Issue by Katana-O** [ultralytics/yolov5#9574]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi dev at YoloV5. I am new to AI industry and YOLOV5.
Allow me to introduce my circumstance and what problem i am facing.

Firstly, I collect 10000 Images and I output a pretrained model, let's call it "pretrained.pt".
[pretrained.pt]
Number of Class: 3
Class Name: ['Dog', 'Cat', 'Horse']
Count of Image: 10000
Estimated time of Training: 2 days.

But now, I realized Cat is not detecting very well. So I collect Another 10000 Images, 5000 comes from Cat, 5000 comes from Lion(Because my Boss ask me to detect more Animal).
I've read your recent post in forum. https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets/38
![image](https://user-images.githubusercontent.com/79778115/192136280-80564ea0-6f7a-40b1-a537-78787a9687f1.png)

This approach tells me to put all images together, and train them, then output a "new.pt". 
[new.pt]
Number of Class:4
Class Name:['Dog', 'Cat', 'Horse', 'Lion']
Count of Image: 20000
Estimated time of Training: 4 days(because 2 days + 2 days).

Point of View regarding to the solution above: 
This is a time costing solution. Let's say in future, if my Boss ask me to detect more different species of animal the time costing will increase more and more(2days + 2days + 2days + 2days, on and on). So I'm asking if there is a better approach to it? Or is there any open-source CV repository that has consider the "update problem"?

**Comment by glenn-jocher** [comment]
@Katana-O this is a common problem, but there's no real 'shortcut' to training additional classes other than to add them to your dataset and retrain. It's possible subsequent retrainings do not require as many epochs as the first training however, as the model has learned many low level features that generalize across classes and domains. 

For info on combining datasets see https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets
